### PR TITLE
HOLD Added docs for org settings

### PR DESCRIPTION
**Pull request by urbanisierung**
New section for Organization Settings under Gettings Started / Camunda Cloud Console

### CODE REVIEW

File: docs/gettingstarted_camunda-cloud-console.md [hunk 1]
```diff
@@ -7,19 +7,57 @@ Camunda Cloud serves as an umbrella for the next generation of Camunda products.
...
 
 When a user signs up to Camunda Cloud, they receive a personal organization. Clusters that the user creates in this organization are assigned to this organization.
 
-If several users need access to the same Zeebe cluster, all users can be assigned to the same organization. A corresponding organisation management for users is planned, but not yet implemented. If it's already necessary for your project, please contact the Camunda Cloud Team.
+If several users need access to the same Zeebe cluster, all users can be assigned to the same organization. If a user is assigned to more than one organization, the organization can be changed in the avatar menu of the navigation bar.
 
-If a user is assigned to more than one organization, the organization can be changed in the avatar menu of the navigation bar.
+## Organization Settings
+
+Organisationseinstellungen können über das Avatar Menu erreicht werden.
```
Comment (ultraschuppi) [inline docs/gettingstarted_camunda-cloud-console.md:27]: görman?

Comment (urbanisierung) [reply docs/gettingstarted_camunda-cloud-console.md]: fixed

File: docs/gettingstarted_camunda-cloud-console.md [hunk 2]
```diff
@@ -7,19 +7,61 @@ Camunda Cloud serves as an umbrella for the next generation of Camunda products.
...
+The provisioning of a Zeebe cluster takes about two minutes. As soon as a Zeebe cluster is operational, its state jumps from _Creating_ to _Healthy_.
 
 To interact with Zeebe, an application must authenticate itself via OAuth. A [client](./zeebecluster_clients.md) must be created in advance for this purpose.
 
 ## Rights concept
 
 When a user signs up to Camunda Cloud, they receive a personal organization. Clusters that the user creates in this organization are assigned to this organization.
 
-If several users need access to the same Zeebe cluster, all users can be assigned to the same organization. A corresponding organisation management for users is planned, but not yet implemented. If it's already necessary for your project, please contact the Camunda Cloud Team.
+If several users need access to the same Zeebe cluster, all users can be assigned to the same organization. If a user is assigned to more than one organization, the organization can be changed in the avatar menu of the navigation bar.
```
Comment (ultraschuppi) [inline docs/gettingstarted_camunda-cloud-console.md:23]: is this even called avatar menu anymore? ;)

Comment (urbanisierung) [reply docs/gettingstarted_camunda-cloud-console.md:23]: How about just removing `avatar`?

**Pull request merged**